src/main/java/at/hannibal2/skyhanni/features/mobs/SpawnTimers.kt

```diff
@@ -0,0 +1,40 @@
+package at.hannibal2.skyhanni.features.mobs
+
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.RenderUtils.drawDynamicText
+import at.hannibal2.skyhanni.utils.TimeUtils
+import net.minecraftforge.client.event.ClientChatReceivedEvent
+import net.minecraftforge.client.event.RenderWorldLastEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SpawnTimers {
+    private var arachneSpawnTime = 0L

```

Comment:
You can use `SimpleTimeMark` perfectly here:
Set it the default variable with `private var arachneSpawnTime = SimpleTimeMark.farPast()`
Check with `arachneSpawnTime.isInPast()` if the timestamp is already past
Use `val countDown = arachneSpawnTime.timeUntil()` to get the time duration until spawn
Use `countDown.format(showMilliSeconds = true)` to get the format as string
and finally, use `SimpleTimeMark.now() + 24.seconds` to set the time to the variable.